enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/database/OnboardingOtpRepository.java

```diff
@@ -41,12 +41,13 @@ public interface OnboardingOtpRepository extends CrudRepository<OnboardingOtpEnt
     Optional<OnboardingOtpEntity> findLastOtp(String processId, OtpType type);
 
     @Modifying
-    @Query("UPDATE OnboardingOtpEntity o SET o.status = com.wultra.app.enrollmentserver.model.enumeration.OtpStatus.FAILED, " +
+    @Query("UPDATE OnboardingOtpEntity o SET " +
+            "o.status = com.wultra.app.enrollmentserver.model.enumeration.OtpStatus.FAILED, " +
             "o.timestampLastUpdated = CURRENT_TIMESTAMP, " +
             "o.errorDetail = 'expired' " +
```

Comment:
Could we use a constant here for the `defined` text? It seems to be same at several other places.

Reply:
Yes, we can use the constant, good point.